src/main/java/frc/robot/subsystems/drive/Swerve.java

```diff
@@ -467,15 +544,44 @@ public Command teleopDriveCommand(
                     rotWeight
             );
 
-            drive(
-                    leftStickSpeeds.getX(),
-                    leftStickSpeeds.getY(),
-                    rot,
-                    true
-            );
+            if (Math.abs(leftStickSpeeds.getNorm()) <= 0.01 && Math.abs(rot) <= 0.01) {
+                wheelX();
```

Comment:
@MqxS  Do we keep this?

Reply:
If yash wants it then yea.

Reply:
Keep it either way we don’t wanna get pushed when we shooting at least